extensions/amp-pinterest/0.1/test/test-amp-pinterest.js

```diff
@@ -34,7 +34,14 @@ describes.realWin(
       );
     }
 
-    function getPin(pinDo, pinUrl, pinMedia, pinDescription) {
+    function getPin(
+      pinDo,
+      pinUrl,
+      pinMedia,
+      pinDescription,
+      width = 20,
```

Comment:
Any reason for this change from 100 ?

Reply:
Thanks for the quick review @wassgha and @kristoferbaxter. I modified the default width and height values in the tests so `this.round` would not default to `true` (and so that the specs could override the dimensions).
 https://github.com/ampproject/amphtml/blob/a4df04761e3e3b69b51f11f8b82c6eeabbd53c6a/extensions/amp-pinterest/0.1/save-button.js#L62-L67.

Reply:
I see! Sounds good! Thank you :)